pages/common/git-scp.md

```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
```

Comment:
```suggestion
```
I don't think this line is necessary as we allow a maximum of 2 lines of page description and 1 for more information link although there are some exceptions. If you think this is an essential line apply the change below.

```suggestion
> Internally this script uses `rsync` and not `scp` as the name suggests.
```